hack/publishing-rules.yaml

```diff
@@ -0,0 +1,98 @@
+skip-godeps: true
+rules:
+- destination: kubernetes-code-generator
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: vendor/k8s.io/code-generator
+- destination: kubernetes-apimachinery
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: vendor/k8s.io/apimachinery
```

Comment:
This won't work well with our workflow during a rebase.  Also, since we don't start from a normal level, you'll have discontinguous commits.